views/interface.js

```diff
@@ -0,0 +1,53 @@
+document.addEventListener('DOMContentLoaded', function(event) {
+    console.log("DOM content loaded")
+
+    getApi()
+    console.log(newsdata)
+});
+
+var newsArray = []
```

Comment:
Is it worth about pulling this out into it's own component, just as we did for `Article`?